setup.py

```diff
@@ -49,7 +49,7 @@
             'flake8',
         ],
         coverage=[
-            'pytest-cov',
+            'coverage',
```

Comment:
@taschini I was researching for some concrete arguments why to use pytest-cov, but found nothing. So if you have some insights please share.